lib/nice_partials/partial.rb

```diff
@@ -16,8 +16,8 @@ class Partial
     #   <%= content.output_buffer %> # => "This line is printed to the `output_buffer`."
     attr_accessor :output_buffer
 
-    def initialize(view_context)
-      @view_context = view_context
+    def initialize(view_context, local_assigns = nil)
```

Comment:
Are there circumstances where the keys in local_assigns aren't symbols? Could this be passed as a double splatted set of keyword arguments?

Reply:
With the updated `render` signature I think there are cases where the `local_assigns` are nil, so I'd prefer to just keep us aware of that here instead of doing any conversion.